text/0000-checkpoints.md

```diff
@@ -17,7 +17,7 @@ Rust's ecosystem, tooling, documentation, and compiler are constantly improving.
 - The standard library and other core ecosystem crates have been updated to use the new features as appropriate.
```

Comment:
.Search and replace has failed us: it needs to be "an epoch" not "a epoch". This happens a few places farther down in the document as well. line 97, line 110, perhaps others (posting via phone unfortunately)